src/EmployerAan/SFA.DAS.EmployerAan/SFA.DAS.EmployerAan.csproj

```diff
@@ -16,4 +17,5 @@
     <ProjectReference Include="..\..\Shared\SFA.DAS.SharedOuterApi\SFA.DAS.SharedOuterApi.csproj" />
```

Comment:
Why was this reference added in here?

Reply:
I did not add this. It was already there in the main branch